README.md

```diff
@@ -108,49 +95,73 @@ Get some help, obviously...
 
 ## What does it do
 
-- Back ups your application settings in a safe directory (e.g. Dropbox)
-- Syncs your application settings among all your workstations
-- Restores your configuration on any fresh install in one command line
-
 By only tracking pure configuration files, it keeps the crap out of your
 freshly new installed workstation (no cache, temporary and locally specific
-files are transfered).
+files are transferred).
 
-Mackup makes setting up the environment easy and simple, saving time for your
-family, great ideas, and all the cool stuff you like.
+Mackup makes setting up the environment easy and simple.
 
-## Bullsh\*t, what does it really do to my files
+There are 2 modes of operations: copy mode and link mode.
+
+### Copy mode
+
+Copy mode is used to back up and restore your files.
+The files are backed up into the configured Mackup folder,
+which can be in Dropbox, iCloud, or wherever you configure it.
+
+It is covered by the 2 commands:
+
+- `mackup backup`
+- `mackup restore`
+
+### Link mode
+
+Link mode is used to move your config files into the Mackup folder,
```

Comment:
What would you think about adding a warning about the fact that the link method is dangerous on macOS Sonoma and later?

Reply:
My bad I somehow missed it